src/disasm.cpp

```diff
@@ -381,19 +388,24 @@ void LineNumberAnnotatedWriter::emitFunctionAnnot(
     }
 }
 
+
 void LineNumberAnnotatedWriter::emitInstructionAnnot(
       const Instruction *I, formatted_raw_ostream &Out)
 {
+    if (ccalls.count(inst_count) > 0) { // there is ccall metadata for this instruction number
+        Out << "; foreign function call to `" << ccalls.lookup(inst_count) << "`\n";
+    } else {
```

Comment:
The itendation of the subsequent block is still off.

Reply:
now fixed it